database/faker/FakerSchoolId.php

```diff
@@ -4,31 +4,15 @@
 
 class FakerSchoolId extends Base
 {
-    /**
-     * A selection of schools from GraphQL
-     *
-     * @var array
-     */
-    protected static $ids = [
-        '3401166',
-        '3401452',
-        '3401451',
-        '3401457',
-        '3401450',
-        '3401460',
-        '4802532',
-        '4811718',
-        '4820400',
-        '4825422',
-    ];
-
     /**
      * Return a random School ID.
      *
      * @return mixed
      */
     public function school_id()
     {
-        return static::randomElement(static::$ids);
+        $ids = file('database/faker/schools.txt');
+
+        return trim($ids[array_rand($ids)]);
```

Comment:
Sorry I feel like I've asked a similar question about `school_id`s before, but why do we need a specific list of ids for this rather than choosing a random string?

Reply:
No need to apologize! It's so we seed with real school ID's, so GraphQL queries like the ones added in #1085 to view school names don't break locally.

Reply:
Ah that does sound super familiar!

Reply:
Added a comment to document! 📝